osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -333,6 +336,16 @@ private void selectTab(int index)
                 ChannelTabControl.Current.Value = channel;
         }
 
+        private void selectChannelSelectorTab()
```

Comment:
I found a better way to write this into ChannelTabControl so wee don't have to do the .Where stuff haha
You can write a public function that just does:
`public SelectChannelSelectorTab() 
{
    SelectTab(selectorTab);
}`